packages/x-charts/src/ChartsTooltip/ChartsItemTooltipContent.tsx

```diff
@@ -23,13 +24,42 @@ export interface ChartsItemTooltipContentProps {
 
 function ChartsItemTooltipContent(props: ChartsItemTooltipContentProps) {
   const { classes: propClasses, sx } = props;
-  const tooltipData = useItemTooltip();
+  const tooltipData = useInternalItemTooltip();
 
   const classes = useUtilityClasses(propClasses);
 
   if (!tooltipData) {
     return null;
   }
+
+  if ('values' in tooltipData) {
```

Comment:
I think [this comment](https://github.com/mui/mui-x/pull/16950#discussion_r2032936062) is still somewhat valid here. 

Would it be easier if `useInternalItemTooltip` always returned an array?

Reply:
> I think this comment is still somewhat valid here.

The returned object shape for radar is now
```js
{
  color,
  label,
 values: [ ... ]
}
```

The previosue solution that returned an array of item was a hack. For example I had to get the `seriesLabel` and color from the first item. Assuming that all have the same series.

Now, the structure is meaningful. We have one series with multiple values. This justifies that this rendering has a `caption` for the root object and a table for the values.